### ISSUE TITLE
ForbiddenComments: Fails to remove empty inline doc block

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#260]
```php
<?php

/** Created by PhpStorm. */
```

With:
```xml
    <rule ref="SlevomatCodingStandard.Commenting.ForbiddenComments">
        <properties>
            <property
                name="forbiddenCommentPatterns"
                type="array"
                value="
                    ~^Created by \S+\.\z~i,
                " />
        </properties>
    </rule>
```
Removes the comment, but `/***/` remains there.
Works for multi-line comments. Likely related to the trailing space in inline comment (which is trimmed).

**Comment by kukulich** [comment]
> Works for multi-line comments.

Strange :) I don't remove any empty doccomment. I may do some other PHPCS sniff...

**Comment by Majkl578** [comment]
Maybe it is TypeHintDeclaration.UselessDocComment? Which does not apply to unbound comments. :)

**Comment by kukulich** [comment]
You're probably right :)

Ok, I'll add really simple remover of empty doccomments to this sniff too.

**Comment by Majkl578** [comment]
Ok, cool, thanks! 👍 Maybe we can consider separate sniff for useless/empty docComments (maybe even extract it from TypeHintDeclaration) in future.

**Comment by kukulich** [comment]
Yop, `TypeHintDeclaration` should be refactored and splitted in next major version (and some other sniffs too).

**Comment by Majkl578** [comment]
Feel free to keep this as a feature request for next major, it's a nitpick mostly. :)

**Comment by kukulich** [comment]
Already working on it ;)

**Comment by kukulich** [comment]
https://github.com/slevomat/coding-standard/commit/5837f9323a3f9424202004d060671ba10215e1b1
